shell/platform/fuchsia/flutter/surface.cc

```diff
@@ -36,7 +36,8 @@ std::unique_ptr<flutter::SurfaceFrame> Surface::AcquireFrame(
       nullptr, std::move(framebuffer_info),
       [](const flutter::SurfaceFrame& surface_frame, SkCanvas* canvas) {
         return true;
-      });
+      },
+      /*frame_size=*/SkISize::Make(800, 600));
```

Comment:
This doesn't look like a unit test so why are we hardcoding it to `(800, 600)`?

Reply:
You are 100% right. Fixed to use the size parameter.